int32 id
BoxObservation[] obs

float32 translation_tolerance
float32 rotation_tolerance
float32 size_tolerance